week02/이성진/todo/src/component/TodoBoard.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import TodoItem from "./Todoitem";
+
+function TodoBoard(props){
```

Comment:
```suggestion
function TodoBoard({todoList}){
```
props를 받고 사용하는 것 보단 구조분해할당을 사용하여 props에 대한 정확한 네이밍을 해주는 것이 좋아보입니다.

Reply:
확인 했습니다!